sensor/common/networkflow/manager/manager_impl_test.go

```diff
@@ -179,7 +189,472 @@ func (suite *NetworkflowManagerTestSuite) TestAddNoOriginator() {
 	h.connectionsSequenceID = sequenceID
 
 	err := h.Process(networkInfoOpen, nowTimestamp, sequenceID)
-	suite.NoError(err)
+	s.NoError(err)
+
+	s.Len(h.endpoints, 1)
+}
+
+// endregion
+
+// region networkFlowManager tests
+
+func (s *NetworkFlowManagerTestSuite) TestEnrichConnection() {
+	mockCtrl := gomock.NewController(s.T())
+	m, mockEntityStore, mockExternalSrc, _ := createManager(mockCtrl)
+	srcID := "src-id"
+	dstID := "dst-id"
+	cases := map[string]struct {
+		connPair                    *connectionPair
+		enrichedConnections         map[networkConnIndicator]timestamp.MicroTS
+		expectEntityLookupContainer expectFn
+		expectEntityLookupEndpoint  expectFn
+		expectExternalLookup        expectFn
+		expectedIndicator           *networkConnIndicator
+		expectedConnection          *connection
+		expectedStatus              *connStatus
```

Comment:
Another bit of a nitpick here, but could we differentiate the prefix from the gomock `EXPECT()` type calls and actual assertions?

Maybe the latter three could have `assert` prefix rather than `expect`? I think the testCase struct would read much better:
```go
connPair                  *connectionPair
enrichedConnections       map[networkConnIndicator]timestamp.MicroTS
expectedCalls             expectFn
assertIndicator           *networkConnIndicator
assertConnection          *connection
assertStatus              *connStatus          
```

Reply:
Skipping for now